tests/csm/rest/test_iam_users.py

```diff
@@ -3925,8 +3943,8 @@ def test_39030(self):
         self.created_iam_users.update({usr_val['user']:usr_val})
         self.log.info("STEP 2: Create another user with same uid and different tenant")
         payload.update({"tenant": "abcc"})
-        payload.update({"access_key": "abcc"})
-        payload.update({"email": "abcc@seagate.com"})
+        del payload["access_key"] 
+        del payload["email"]
```

Comment:
Is this required?

Reply:
These statements:
payload.update({"access_key": "abcc"})
payload.update({"email": "abcc@seagate.com"})

are not required as per they are hardcoded and causing conflict(409) while creating new user with same email and access_key, hence forth removed the same.

cc @kachhwahadivya